go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/go-openapi/errors v0.20.4
 	github.com/go-openapi/strfmt v0.21.7
 	github.com/go-openapi/swag v0.22.9
-	github.com/go-playground/validator/v10 v10.13.0
+	github.com/go-playground/validator/v10 v10.15.5
```

Comment:
Did you maybe use a `go get...` command to bump the assisted-image-service dependency? Because it looks like some other unrelated modules were bumped, such as the `validator`.
Can you please try instead to:
* Manually edit the go.mod just for `github.com/openshift/assisted-image-service v0.0.0-20231130154654-56d03d1503e2`
* Then run a `go mod tidy` and a `go mod vendor`

I'd expect to have just a couple of files updated (and no other dependencies). This will a safer approach for this case, since just the very minimum required will be updated.

Reply:
Yes I did use go get to bump the version. Sure, I will just change the required module

Reply:
@andfasano , Just updated the assisted-image-service module and did vendor the same now. Please re-review